lib/mix/lib/mix/tasks/release.init.ex

```diff
@@ -217,11 +219,9 @@ defmodule Mix.Tasks.Release.Init do
     """
 
   @doc false
-  def start_bat_text,
+  def env_bat_text,
```

Comment:
Why the `env.bat` does not contain the heart instructions? 🙂

Reply:
Heart is not available on Windows. Instead one would use Windows Services.

Reply:
Oh! I thought it was supported because the documentation says:

> To reboot on Windows, `HEART_COMMAND` can be set to `heart -shutdown` (included in the Erlang delivery) or to any other suitable program that can activate a reboot.